presentation/handler/user_handler.go

```diff
@@ -86,32 +86,31 @@ func (h UserHandler) Delete(c *fiber.Ctx) error {
 	return core.Success(c, fiber.StatusOK, SuccessMsgUserDelete, nil)
 }
 
-// Register 	parses a dto.UserInputDTO from the request body, compares and validates both passwords and adds a new user to the userStorage.
+// Create 	parses a dto.UserInputDTO from the request body, compares and validates both passwords and creates a new user.
 //
-//	@Summary	Register User
+//	@Summary	Create User
 //	@Tags		User
 //	@Accept		json
 //	@Produce	json
 //	@Param		request	body		dto.UserInputDTO	true	"Request Body"
-//	@Success	200		{object}	dto.GeneralResponseDTO{data=dto.UserDetailedOutputDTO}
-//	@Router		/api/user/register [post]
-func (h UserHandler) Register(c *fiber.Ctx) error {
+//	@Success	200		{object}	dto.GeneralResponseDTO{data=dto.UserCoreOutputDTO}
+//	@Router		/api/user [post]
+func (h UserHandler) Create(c *fiber.Ctx) error {
```

Comment:
Can we keep the name register for that method? I think the name register is fine for a handler method.